src/pages/index.js

```diff
@@ -144,10 +144,10 @@ const StyledWalletConnectLogo = styled.div`
 
 const StyledWalletConnectButton = ConnectButton.extend`
   &:hover {
-    background: #454852;
+    background: #4fa1ff;
```

Comment:
@cipherpeon quick thought here: i would think eventually these values could be added to the `/src/styles.js` in an object for each named color like dis

```js
{
  walletconnect: {
    light: '...',
    mid: '...',
	dark: '...'
  },
}
```

Probably all using the css color function to do lightness / darkness adjustments based on a percentage value to keep the color shifts unique between each state for buttons as well as elements.

Thoughts @mikedemarais ?

Reply:
actually looks like this color was fixed in this commit https://github.com/balance-io/balance-manager/commit/880ff676b45637859d041f94588b0ffc31422dd2. could make sense to rebase from this and use those colors to reduce colors not kept in variables?